### ISSUE TITLE
Q: Better way to set the stylesheet attribute?

### ISSUE THREAD

**Issue by EmilStenstrom** [xflr6/graphviz#173]
Hi, this might be a question, not a feature request, but I've googled and checked the docs to no avail.

Is it possible to set the [stylesheet](https://graphviz.org/docs/attrs/stylesheet/) attribute of a graph in a nicer way? I've gotten it working with:

```python
dot = Digraph(
    format='svg',
    body=['\tstylesheet="/static/graph.css"\n'],
)
dot.attr('node', shape='rect', style='filled', fillcolor="white")
```

To make the output look nice I have to add the whitespace myself, and since the stylesheet declaration should be first, I need to move the node_attr out to a separate call afterwards. 

**Is there a nicer way to do this?**

**Comment by xflr6** [comment]
Hi. Yes, use the `.attr()`-method (see https://graphviz.readthedocs.io/en/stable/manual.html#attributes):

```python
import graphviz

dot = graphviz.Digraph(format='svg')

dot.attr(stylesheet='spam.css')

dot.attr('node',shape='rect', style='filled', fillcolor='white')

print(dot.source)
```

```
digraph {
	stylesheet="spam.css"
	node [fillcolor=white shape=rect style=filled]
}
```

**Comment by EmilStenstrom** [comment]
Thank you! I will read the docs more thoroughly next time! I got as far as `dot.attr('stylesheet', 'spam.css')` but that doesn't obviosly work!

I also want to thank you for putting down the time on this project! Open source is thankless work, unless someone actually thanks you sometime! :D

**Comment by xflr6** [comment]
Thanks so much :)

I have noticed that the links from the  API docs to the user  guide where not working, fixed (in latest) via 37212024253a685c68d57b45d2bcbb6e6992f4dd and d05ed193a6ef1374e5d0c356704ad38534abacc9.
